lib/watir-webdriver/browser.rb

```diff
@@ -10,6 +10,7 @@ class Browser
     include Waitable
 
     attr_reader :driver
+    attr_reader :checkers
```

Comment:
"error_checkers" is kind of ambiguous, and "run_checkers" even moreso, so I think doing just "checkers" is worse, especially since it allows you to do anything you want after a potential page load. I agree with @jkotests, let's rename everything to AfterHooks, and allow an alias for Browser#checkers